AerospikeClient/Main/Info.cs

```diff
@@ -579,14 +580,13 @@ private string ParseSingleResponse(string name)
 		public void ParseName(string name)
 		{
 			int begin = offset;
+			ReadOnlySpan<byte> nameBytes = MemoryMarshal.AsBytes(name.AsSpan());
```

Comment:
This does not account for name being UTF8 encoded.

Reply:
Good catch. I think what could be done here it to pass the name as utf8 bytes here but unfortunately this method is in the public interface so I won't change it for now.